src/main/java/org/broadinstitute/hellbender/tools/funcotator/FilterFuncotations.java

```diff
@@ -141,11 +147,30 @@ private void registerFilters() {
         funcotationFilters.add(new ClinVarFilter(afDataSource));
         funcotationFilters.add(new LofFilter(reference, afDataSource));
         funcotationFilters.add(new LmmFilter());
+        funcotationFilters.add(new ArHomvarFilter());
     }
 
     @Override
-    public void apply(final VariantContext variant, final ReadsContext readsContext, final ReferenceContext referenceContext, final FeatureContext featureContext) {
-        outputVcfWriter.add(applyFilters(variant, getMatchingFilters(variant)));
+    public void firstPassApply(final VariantContext variant, final ReadsContext readsContext, final ReferenceContext referenceContext, final FeatureContext featureContext) {
+        buildArHetByGene(variant);
+    }
+
+    @Override
+    protected void afterFirstPass() {
+        arHetVariantsByGene.keySet().forEach(gene -> {
+            if(arHetVariantsByGene.get(gene).size() > 1) {
+                arCompoundHetVariants.addAll(arHetVariantsByGene.get(gene));
+            }
+        });
+    }
+
+    @Override
+    public void secondPassApply(final VariantContext variant, final ReadsContext readsContext, final ReferenceContext referenceContext, final FeatureContext featureContext) {
+        Set<String> matchingFilters = getMatchingFilters(variant, funcotationFilters);
```

Comment:
`matchingFilters` can be `final`.

In general for GATK tools, we try to make everything `final` if it can be.